engine/release-notes/23.0.md

```diff
@@ -41,6 +41,33 @@ Changing the version format is a stepping-stone towards Go module compatibility,
 but the repository doesn't yet use Go modules, and still requires using a "+incompatible" version.
 Work continues towards Go module compatibility in a future release.
 
+## 23.0.4
+
+{% include release-date.html date="2023-04-17" %}
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 23.0.4 milestone](https://github.com/docker/cli/milestone/77?closed=1)
+- [moby/moby, 23.0.4 milestone](https://github.com/moby/moby/milestone/117?closed=1)
+
+### Bug fixes and enhancements
+
+- Fix a performance regression in Docker CLI 23.0.0 [docker/cli#4141](https://github.com/docker/cli/pull/4141).
```

Comment:
Is the regression fully fixed or just improved?

Reply:
It should be fully fixed; benchmarks can be found here; https://github.com/docker/cli/pull/4173#issuecomment-1503842033